### ISSUE TITLE
Modal does not have a focus trap

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#600]
It should not be possible to reach elements outside of a modal with the tab key.

**Comment by Arnei** [comment]
Fixed by #1339
